src/stores/journeyActions.js

```diff
@@ -29,25 +33,106 @@ export function createJourneyPath(journey) {
 }
 
 export default (state) => {
+  function getJourneyFromStateAndTime(time) {
```

Comment:
Normally, a "journey" object is an hfp item with only the props that identify the journey present. In other words, it is based on data that is fetched. With this function we can create a composite journey object from a departure time without fetching hfp data if we know the route and the date.